dev/reference/key-monitoring-metrics/tidb-dashboard.md

```diff
@@ -8,7 +8,7 @@ category: reference
 
 If you use Ansible to deploy the TiDB cluster, the monitoring system is deployed at the same time. For more information, see [TiDB Monitoring Framework Overview](/how-to/monitor/overview.md).
 
-The Grafana dashboard is divided into a series of sub dashboards which include Overview, PD, TiDB, TiKV, Node\_exporter, Disk Performance, and so on. A lot of metrics are there to help you diagnose.
+The Grafana dashboard is divided into a series of sub dashboards which include Overview, PD, TiDB, TiKV, Node\_Exporter, Disk Performance, and so on. The TiDB dashboard consists of the TiDB panel and the TiDB Summary panel, and what displays on the TiDB Summary panel also displays on the TiDB panel. A lot of metrics are there to help you diagnose.
```

Comment:
> I remember that `Node_exporter` is exactly what's displayed on the dashboard.

When @liubo0127 was reviewing the [TiDB Cluster Alerting Metrics Guide - Google 文档](https://docs.google.com/document/d/1yiv4yRWQ-skuHp1RcagDCMt8bFqSmkDb4I1gdBsS_JU/edit#), he suggested I use "Node_Exporter" instead of "Node_exporter". Please help confirm this name usage. @lonng If `Node_exporter` is what's displayed on the dashboard, I think we should consistently use it in our documents.

Reply:
Should I use "Node_Exporter" or "Node_exporter"? PTAL @lonng

Reply:
The name of it in Grafana dashboard is `Node_exporter`, it's better to keep them consistent.

Reply:
Got it, thanks~